### PR TITLE
feat(admin,core,router): ping routes

### PR DESCRIPTION
**Pull request by kon14**
Introduces a ping route for Conduit's administrative APIs to be used by user input checks in CLI.
Removes legacy ping routes.

**What kind of change does this PR introduce?** 

- [ ] Bugfix
- [X] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other (please describe)

**Does this PR introduce a breaking change?** 

- [X] Yes
- [ ] No

**The PR fulfills these requirements:**

- [X] It's submitted to the `main` branch
- [ ] When resolving a specific issue, it's referenced in the PR's description  (e.g. `fix #xxx`, where "xxx" is the issue number)

If adding a **new feature**, the PR's description includes:

- [X] A convincing reason for adding this feature

### CODE REVIEW

File: packages/core/src/routes/index.ts [hunk 1]
```diff
@@ -1,18 +1,12 @@
 import ConduitGrpcSdk, {
   ConduitRouteActions as Actions,
```
Comment (Michael-Vol) [inline packages/core/src/routes/index.ts]: Remove import alias

File: packages/core/src/routes/index.ts [hunk 2]
```diff
@@ -1,18 +1,12 @@
...
   ConduitRouteReturnDefinition,
```
Comment (Michael-Vol) [inline packages/core/src/routes/index.ts]: This import is not being used anywhere, probably a leftover

File: packages/core/src/routes/index.ts (hunk 1 above)
Comment (kon14) [reply packages/core/src/routes/index.ts]: You're too slow for this kid.

File: packages/core/src/routes/index.ts (hunk 2 above)
Comment (kon14) [reply packages/core/src/routes/index.ts]: 👴

**Review by Michael-Vol — APPROVED** [review]

**Review by ChrisPdgn — APPROVED** [review]

File: packages/core/src/routes/index.ts [hunk 3]
```diff
@@ -21,34 +15,9 @@ export class HttpServer {
   }
 
   private registerRoutes() {
```
Comment (kkopanidis) [inline packages/core/src/routes/index.ts:17]: if everything is commented out, remove the registration on the bottom, don't leave it there. It's probably for the best to remove this entire file, it's not needed without these routes. Also removal of the "waitForExistence" that concerns the router, is required

File: packages/admin/src/routes/Ready.ts [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+export function getReadyRoute() {
+  return new ConduitRoute(
+    {
+      path: '/ready',
+      action: ConduitRouteActions.GET,
+    },
+    new ConduitRouteReturnDefinition(`Ready`, {}),
+    async () => {
+      return 'Conduit Admin is online!';
```
Comment (kkopanidis) [inline packages/admin/src/routes/Ready.ts:12]: core is online

File: packages/admin/src/middleware/Auth.middleware.ts [hunk 5]
```diff
@@ -17,6 +17,7 @@ export function getAuthMiddleware(grpcSdk: ConduitGrpcSdk, conduit: ConduitCommo
       req.originalUrl.indexOf('/graphql') === 0 && req.method === 'GET';
     if (
```
Comment (kkopanidis) [inline packages/admin/src/middleware/Auth.middleware.ts:18]: can we move the exclusions to an array and add a cleaner check? It's getting too big at this point.

**Review by kkopanidis — CHANGES_REQUESTED** [review]

**Review by kkopanidis — APPROVED** [review]

**Pull request merged**